mbof/views.py

```diff
@@ -42,16 +44,28 @@ class CurrentUserViewSet(UserViewSet):
     queryset = User.objects.filter(loginName=os.getenv('REMOTE_USER')).order_by('surname')
 
 
-class MessageViewSet(viewsets.ModelViewSet):
+class EventViewSet(viewsets.ModelViewSet):
     """
     API endpoint that allows users to be viewed or edited.
     """
-    queryset = Message.objects.all().order_by('postingTime')
-    serializer_class = MessageSerializer
+    queryset = Event.objects.all().order_by('postingTime')
+    serializer_class = EventSerializer
+
+
+class CurrentEventViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows users to be viewed or edited.
+    """
+    serializer_class = EventSerializer
+
+    def get_queryset(self):
+        current_time = timezone.now()
+        return Event.objects.filter(startTime__lte=current_time).filter(endTime__gte=current_time)
```

Comment:
Be very careful to understand exactly how time and timezones affect the application.  For instance don't assume that the client and server are in the same timezone, don't assume that the server hasn't changed it's timezone (certainly possible with cloud deployments), make sure to account for daylight savings time.